### DISCUSSION TITLE
Using system libs instead of compiling port dependencies

### DISCUSSION THREAD

**Discussion by srebrnyp** [microsoft/vcpkg#30340] [Q&A] [answered]
Hi,

This is general question but there is example for my use case:

I want to use **bzip2** library installed on the system while installing **poco** library.
**poco** has dependency to **pcre2** and this has dependency to **bzip2**
But i dont want to install additional port for bzip2 library, i want to use bzip2 library that i have installed on the system. How to do it (manifest mode)?

Best Paweł

**Comment by Neumann-A** [comment, marked as answer]
You provide an empty port overlay for bzip2 (or one which checks if the system really has it). Vcpkg will still 'install' bzip2 but it will be a 'nop' since nothing gets installed.
